voting/ethereum/instructions/guardian_registration.md

```diff
@@ -0,0 +1,81 @@
+in the test box labeled *ABI / JSON Interface*. 
+![](../instructions/enroll_guardian_2.png)
+1. Click *Access*.
+* If the *Access* button appears disabled, make sure there are no trailing line feeds at the bottom of the *ABI / JSON Interface* text box.
+1. Select `register` in the drop down list *Read / Write Contract*
+![](../instructions/enroll_guardian_3.png)
+The function parameters for the `register` call will appear in the form.
+1. Fill the fields labeled `name` and `website` 
+with your guardian details.
+![](../instructions/enroll_guardian_4.png)
+1. Choose one of the options under *How would you like to access your wallet?*
+and provide your wallet information/credentials.
+![](../instructions/enroll_guardian_5.png)
+1. Proceed to *Select an Address*. Select your Guardians's Ethereum address. 
+The address you choose will be the Guardians's identification for purpose
+ of delegation later on.
+![](../instructions/enroll_guardian_6.png)
+Click *Unlock*
+1. After unlocking an account, a new field labeled *Value* will appear after the website entry field (see below).
+Enter `1` in the *Value* field. This will send 1 Ether with your registration request as a deposit. `OrbsGuardian` contract
+will refund you the deposit if you later request to unregister.
+A 1 Ether deposit is required for registration. You may not send more than exactly 1 Ether.
+![](../instructions/enroll_guardian_7.png)
+1. Verify that the option *Automatically Calculate Gas Limit* is switched off, and manually adjust the *Gas Limit* field as required. `500,000` will be more than enough (at the time this is being written a successful registration uses ~115000 Gas).
+![](../instructions/enroll_guardian_8.png)
+Click *Write* 
+![](../instructions/enroll_guardian_9.png)
+Click *Sign Transaction*
+![](../instructions/enroll_guardian_10.png)
+Click *Send Transaction*
+![](../instructions/enroll_guardian_11.png)
+Review, then click *Send*
+1. Once the transaction is sent, MyCrypto will provide a link to track the transaction status on Etherscan.
+Navigate to *Etherscan* by clicking *Verify (Etherscan)*
+![](../instructions/enroll_guardian_12.png)
+
+1. Confirm the transaction has been accepted successfully.
+![](../instructions/etherscan_confirmation_2.png)
+ 
+Make sure you see 
+> TxReceipt Status:Success
+
+With sufficient block confirmations.
+
+[1]: https://ropsten.etherscan.io/address/0x71715337C81a99F1B02c3467168d5d657CeE6bfc#code
+
+##### Notes
+* The contract does not enforce uniqueness of name and website values between the different Guardians. Two Guardians may register with same name, or website url. It is up to the Delegators to vet their Guardians. 
+* To make changes to your registration repeat the process providing new values. A deposit is not required when re-registering. If you send a second deposit the transaction will revert.
+* To to be removed from the Guardians list, use the leave() function.
```

Comment:
consider adding "use the `leave` function, signing the transaction with the guardian account you wish to unregister." for clarity